### DISCUSSION TITLE
Repeater format more compact (like KeyValue)

### DISCUSSION THREAD

**Discussion by pedrocoelho98** [filamentphp/filament#3429] [Help] [answered]
Hello.

Is there anyway to format the Repeater component more like the KeyValue, i.e. remove the borders and present the items in a more compact way?

instead of this:
![image](https://user-images.githubusercontent.com/45846855/183457320-6ab7e1dc-b364-43e2-8447-bbd847d6137c.png)

something like this:
![image](https://user-images.githubusercontent.com/45846855/183459512-603f002a-6612-45e9-8ed1-b99f9970fb10.png)

**Comment by danharrin** [comment, marked as answer]
There isn't really a way to move the labels like that, but you can remove the containers to make it more compact

Copied the `repeater.blade.php`, removed all the container padding, and then saved it in a Blade file like `views/forms/compact-repeater.blade.php`. And then pass in `->view('forms.compact-repeater')` to the Repeater when you use it.

However, a new `->compact()` method for the repeater that modifies the existing layout for you would be nice. Want to PR?

**Reply by pedrocoelho98** [reply]
Thanks!
That puts me in the right pad.

I'd love to PR but I'm still as green as they come from the nursery!
I'll try my best, but everyone here is more capable for that than me

**Reply by danharrin** [reply]
Check this out - https://github.com/icetalker/filament-table-repeater

Maybe you could use that instead

**Reply by pedrocoelho98** [reply]
That is exactly what I was looking for! Thank you so much!

> Check this out - https://github.com/icetalker/filament-table-repeater
> 
> Maybe you could use that instead

**Reply by myregistration** [reply]
It would be nice if we could use ->contained(false) like we can with the Infolist RepeatableEntry

**Comment by ramirobrescia** [comment]
Hi every one.

I make a theme and set this css hook classes:

```css
/* Repeater compact */
.fi-fo-repeater-item {
	@apply flex divide-none;
}

.fi-fo-repeater-item-header {
	@apply order-last p-3;
}

.fi-fo-repeater-item-content {
	@apply p-3;
}
```

And move from this...

![imagen](https://github.com/user-attachments/assets/dab22ccd-4028-4c07-8336-ee4c825eb228)

To this...

![imagen](https://github.com/user-attachments/assets/09f58a7c-195b-4b42-b993-9c651ee56cf6)


Also if we hide labels and set a label as placeholder

```php
Select::make('product_id')
    ->hiddenLabel()
    ->placeholder(__('Product'));
```
Looks like this...

![imagen](https://github.com/user-attachments/assets/afb211cc-e24c-4598-a090-335e818c25d5)

Thank you. A hug!
